src/components/TasksTable/index.jsx

```diff
@@ -254,6 +315,46 @@ export default class TasksTable extends Component {
             onFilterClick={this.handleFilterToggle}
           />
         </div>
+        <Drawer
+          anchor="right"
+          open={drawerOpen}
+          onClose={this.handleDrawerClose}>
+          <div className={classes.drawer}>
+            <div className={classes.drawerHeader}>
+              <IconButton onClick={this.handleDrawerClose}>
```

Comment:
Have this button positioned to the right of "Summary" to make sure we don't waste vertical space.

<img width="366" alt="screen shot 2018-07-06 at 7 42 27 pm" src="https://user-images.githubusercontent.com/3766511/42404143-ca3be522-8154-11e8-8612-e400dac4fee9.png">